docker-compose.production.yml

```diff
@@ -18,9 +20,9 @@ services:
     depends_on:
       - postgres
       - mongo
-      - redis
```

Comment:
@geoffturk Have we deployed this to production? I don't know why the production Docker Compose does not have Redis set up. Do we use a managed Redis server?

Reply:
@maxsarratt This has never been deployed in production, which is why we only have a `dev` branch and no `main`. We should add Redis to Docker Compose if we need it.